.buildkite/pipeline.build.yml

```diff
@@ -573,6 +573,7 @@
     - pip uninstall -y ray && rm -rf /ray/python/ray/thirdparty_files
     - pushd /ray && git clean -f -f -x -d -e .whl -e python/ray/dashboard/client && popd
     - bazel clean --expunge
+    - python ./ci/env/setup_credentials.py wandb
```

Comment:
Note that this will fail on PR builds. Can we somehow skip the test in that case?

Reply:
Could we add credentials to PR builds?

Once we make all code snippets runnable, we'll need credentials for other docstring examples.

Reply:
It's not straightforward as anyone could file a PR that exposes the secrets. This is why we have different permissions set-up for branch builds (e.g. master) and PRs - we do control what we merge, but we don't control the PRs anyone can open.